common/source/docs/common-oem-customizations.rst

```diff
@@ -6,53 +6,63 @@ OEM Customization
 
 ArduPilot provides several ways for an OEM to provide firmware on their products that has been customized to their specific system configuration:
 
-- Ability to have specific defaults for parameters to match included system components, such as gimbals, or entire systems' peripherals, such as in ready-to-fly vehicles. This allows the user to "reset to defaults" using Mission Planner or MAVProxy in case of a problem with parameters having been accidentally changed, and only have to re-calibrate compass/IMU/and RC, at most, in order to be ready to fly.
+- Ability to have specific defaults for parameters to match included system components, such as gimbals, or entire systems' peripherals, such as in ready-to-fly vehicles. This allows the user to "reset to defaults" using Mission Planner or MAVProxy in case of a problem with parameters having been accidentally changed, and only have to re-calibrate compass/IMU/and RC, at most, to be ready to fly.
 - Ability to provide Lua scripts in ROM for special functions without the end-user having to load them on the SD card. Read more about Lua scripting in ArduPilot :ref:`here<common-lua-scripts>`.
+- Ability to change the firmware string displayed to the user.
 - Ability to change parameters and mark them as read-only so that users cannot change them using :ref:`APJ Tools<dev:apjtools-intro>`.
 
 Customization Steps
 ===================
 
-This assumes that the OEM has set up the build environment (:ref:`building-the-code`) and cloned the ArduPilot GitHub repo locally (:ref:`where-to-get-the-code`), in order to build its customized version of the firmware.
+This section assumes that the OEM has set up the build environment (:ref:`building-the-code`) and cloned the ArduPilot GitHub repo locally (:ref:`where-to-get-the-code`), to build its customized version of the firmware.
 
-1. Create a branch with the version of firmware you wish to base the customization upon. This will usually be the current stable version. To do this for ArduPlane Stable, for example, assuming you are already in the ArduPilot directory on your PC:
+#. Create a branch with the version of firmware you wish to base the customization upon. This will usually be the current stable version. To do this for ArduPlane Stable, for example, assuming you are already in the ArduPilot directory on your PC:
 
-::
+    .. code-block:: bash
 
-    git fetch https://github.com/ArduPilot/ardupilot.git <version>
-                           where <version> is the tag for the stable version for the
-                           desired vehicle: ArduPlane-stable, ArduCopter-stable,
-                           APMrover2-stable,etc.
+        git fetch https://github.com/ArduPilot/ardupilot.git <version>
+                            where <version> is the tag for the stable version for the
+                            desired vehicle: ArduPlane-stable, ArduCopter-stable,
+                            APMrover2-stable,etc.
```

Comment:
```suggestion
                            desired vehicle: Plane-4.2, Copter-4.2,
                            Rover-4.2, etc.
```

Reply:
The stable tags will always be relevant rather than having to update this page for every feature release.

Reply:
yeah... they didnt show up on autocomplete and I forgot APMrover is still a thing
you can tell which tags i look for LOL